ouroboros-consensus/docs/tutorials/Ouroboros/Consensus/Tutorial/Simple.lhs

```diff
@@ -578,6 +579,9 @@ If it did, that means the `LedgerState` is such that it is in a state that will
 eventually fail due to the passage of time and such errors should have been
 signalled earlier (for example, when applying blocks.)
 
+The `converMapKind` function is part of the UTxO-HD feature and for the sake of
+this tutorial it shall be ignored. There is an appendix at the end of the
+tutorial that describes UTxO-HD in broad terms.
```

Comment:
As a catch-all comment for UTxO-HD related features, maybe we can mention at the start of the ledger state section that anything related to mapkinds and ledger tables will be ignored until the appendix. This also makes clear that we won't explain what `ValuesMK` and `DiffMK` mean in the definition of `applyBlockTo`, and why the `ApplyBlock` instance has a `getBlockKeySets` definition. Or, we should explicitly point out the UTxO-HD related features each time one comes up in the code snippets